app/views/list/list.xml

```diff
@@ -1,87 +1,87 @@
-				<activity-indicator busy="{{ isHistoryLoading }}"
-					rowSpan="3" />
-			</grid-layout>
-		</drawer:rad-side-drawer.drawerContent>
-
-	</drawer:rad-side-drawer>
+<page loaded="loaded" xmlns="http://schemas.nativescript.org/tns.xsd" xmlns:drawer="nativescript-telerik-ui/sidedrawer" xmlns:lv="nativescript-telerik-ui/listview" xmlns:FabButton="nativescript-floatingactionbutton/fab">
+	<page.actionBar>
+		<action-bar title="Groceries">
+			<action-bar.actionItems>
+				<action-item text="Sign out" tap="signOut" ios.position="left" android:icon="res://ic_logout" />
+				<action-item android.text="Share" tap="share" ios:icon="res://share" android:icon="res://ic_share" ios.position="right" />
+			</action-bar.actionItems>
+		</action-bar>
+	</page.actionBar>
+
+	<drawer:rad-side-drawer id="drawer" drawerLocation="Right">
+		<drawer:rad-side-drawer.drawerTransition>
+			<drawer:PushTransition />
+		</drawer:rad-side-drawer.drawerTransition>
+
+		<drawer:rad-side-drawer.mainContent>
+			<grid-layout rows="auto, *" id="main-content">
+				<grid-layout row="0" columns="*, auto" class="main-color">
+					<text-field id="grocery" text="{{ grocery }}" hint="Enter a grocery item" returnKeyType="done" returnPress="add" col="0" />
+					<Image ios:src="res://history" android:visibility="collapsed" tap="history" col="1" />
+				</grid-layout>
+ 
+				<lv:rad-list-view
+					items="{{ groceryList }}"
+					id="grocery-list"
+					pullToRefresh="true"
+					pullToRefreshInitiated="pullToRefreshInitiated"
+					selectionBehavior="None"
+					swipeCells="true"
+					startSwipeCell="startSwipeCell"
+					row="1"
+					rowSpan="2">
+					<lv:rad-list-view.listViewLayout>
+						<lv:ListViewLinearLayout scrollDirection="Vertical" itemHeight="50" />
+					</lv:rad-list-view.listViewLayout>
+					<lv:rad-list-view.itemTemplate>
+						<grid-layout columns="auto, *" backgroundColor="white">
+							<image tap="{{ toggleDone }}" src="{{ done ? 'res://checked' : 'res://unchecked' }}" class="checkbox" col="0" />
+							<label text="{{ name }}" col="1" />
+						</grid-layout>
+					</lv:rad-list-view.itemTemplate>
+					<lv:rad-list-view.itemSwipeTemplate>
+						<grid-layout columns="auto, *, auto">
+							<stack-layout class="swipe-image-container done-button-container" col="0">
+								<image ios:src="res://done" android:src="res://ic_done" tap="{{ toggleDone }}" class="swipe-button" />
+							</stack-layout>
+							<Placeholder col="1" />
+							<stack-layout class="swipe-image-container delete-button-container" col="2">
+								<image ios:src="res://delete" android:src="res://ic_delete" tap="{{ swipeDelete }}" class="swipe-button" col="2" />
+							</stack-layout>
+						</grid-layout>
+					</lv:rad-list-view.itemSwipeTemplate>
+				</lv:rad-list-view>
+
+				<android>
+                    <FabButton:fab row="1" tap="history" icon="ic_history" backColor="#336699" rippleColor="#999" class="fab-button" margin="15" />

```

Comment:
Can the `margin` be moved to the existing `.fab-button` selector in the CSS file?

`backColor` would be nice to move too, but I’m guessing that one needs to be in the XML.